From groeck@redback.com  Mon Aug  2 16:30:02 2010
From: Guenter Roeck <guenter.roeck@ericsson.com>
To: "James E.J. Bottomley" <James.Bottomley@suse.de>,
	Richard Purdie <rpurdie@rpsys.net>,
	Greg Kroah-Hartman <gregkh@suse.de>,
	Benjamin Herrenschmidt <benh@kernel.crashing.org>,
	Paul Mackerras <paulus@samba.org>
Cc: Jean Delvare <khali@linux-fr.org>,
	Dmitry Torokhov <dtor@mail.ru>, Liam Girdwood <lrg@slimlogic.co.uk>,
	Mark Brown <broonie@opensource.wolfsonmicro.com>,
	Jani Nikula <ext-jani.1.nikula@nokia.com>,
	Linus Walleij <linus.walleij@stericsson.com>,
	Chris Wright <chrisw@sous-sol.org>,
	Nick Cheng <nick.cheng@areca.com.tw>,
	James Smart <james.smart@emulex.com>,
	Alex Iannicelli <alex.iannicelli@emulex.com>,
	"Eric W. Biederman" <ebiederm@aristanetworks.com>,
	Benjamin Thery <benjamin.thery@bull.net>, Tejun Heo <tj@kernel.org>,
	linux-kernel@vger.kernel.org, linux-scsi@vger.kernel.org,
	linuxppc-dev@ozlabs.org, Guenter Roeck <guenter.roeck@ericsson.com>
Subject: scsi: Remove owner field from attribute initialization in LPFC driver
Date: Wed, 28 Jul 2010 22:09:21 -0700
Message-ID: <1280380166-29196-2-git-send-email-guenter.roeck@ericsson.com>

Signed-off-by: Guenter Roeck <guenter.roeck@ericsson.com>
Acked-by: Tejun Heo <tj@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/scsi/lpfc/lpfc_attr.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/scsi/lpfc/lpfc_attr.c
+++ b/drivers/scsi/lpfc/lpfc_attr.c
@@ -2720,7 +2720,6 @@ static struct bin_attribute sysfs_drvr_s
 	.attr = {
 		.name = "lpfc_drvr_stat_data",
 		.mode = S_IRUSR,
-		.owner = THIS_MODULE,
 	},
 	.size = LPFC_MAX_TARGET * MAX_STAT_DATA_SIZE_PER_TARGET,
 	.read = sysfs_drvr_stat_data_read,
